libr/debug/debug.c

```diff
@@ -1561,7 +1561,17 @@ static int show_syscall(RDebug *dbg, const char *sysreg) {
 	return reg;
 }
 
-// continue execution until a syscall is found, then return its syscall number or -1 on error
+/* Continue execution until a syscall is found.
+ *
+ * If no plugin is selected, read the x86 sn register in order to determine the syscall. See https://www2.lauterbach.com/pdf/debugger_x86.pdf.
```

Comment:
i dont see whats the relationship between this pdf of > 100 pages and this sentence.

Reply:
the pdf document how works the sn register in Linux.

the debugger method is very obscure!!!